Working on Issue1

shikha8997 opened the pull request:
Closes #1

abilashravi-ta reviewed (changes requested):
@shikha8997, please make the following changes:

- Add an appropriate [PR description](https://github.com/shikha8997/mle-training/pull/2#issue-1179502688) as mentioned in the _assignment PS_.
- Add the `env.yml` file to the _branch_.
- Remove unnecessary files (_test.py_ in this case).

shikha8997 commented:
@abilashravi-ta, I have made the following changes:
1. Added an appropriate PR description and linked it to Issue1.
2. env.yml is added to main branch.
3. Removed unnecessay files (test.py)
